src/fragments/lib-v1/datastore/native_common/getting-started.mdx

```diff
@@ -113,10 +113,6 @@ import android13 from '/src/fragments/lib-v1/datastore/android/getting-started/4
 
 <Fragments fragments={{ android: android13 }} />
 
-import flutter14 from '/src/fragments/lib-v1/datastore/flutter/getting-started/40_codegen.mdx';
```

Comment:
Thanks for removing this. This looks like a miss from when Amplify Flutter v1 was added to the docs. The file is not unused with this reference removed. Can you remove the file as well?